config/routes.rb

```diff
@@ -91,13 +91,17 @@ def scoped_navigation_routes(context, navigation)
 
     # Routes to support a simple URL for mailings from Benefits Data Trust (BDT) to their clients
     # And for 211 hotline referrals to both full service and EIP
+    # And for FreshEBT (aka Propel) for in-app referrals to EIP
     # These routes point people to the desired landing pages with the proper source tracking
     # They can be removed after 10/15/20. When these routes are removed they'll go to the homepage so won't be broken.
     get "/bdt", to: "public_pages#eip_home", defaults: {"s": "bdt"}
     get "/BDT", to: "public_pages#eip_home", defaults: {"s": "bdt"}
     get "/211", to: "public_pages#full_service_home", defaults: {"s": "211"}
     get "/211-stimulus", to: "public_pages#eip_home", defaults: {"s": "211-stimulus"}
     get "/211-Stimulus", to: "public_pages#eip_home", defaults: {"s": "211-stimulus"}
+    get "/FreshEBT", to: "public_pages#eip_home", defaults: {"s": "propel-eip"}
+    get "/freshEBT", to: "public_pages#eip_home", defaults: {"s": "propel-eip"}
+    get "/freshebt", to: "public_pages#eip_home", defaults: {"s": "propel-eip"}
```

Comment:
99.9% of the people visiting will just use the first link that Propel will embed in their app. I'm just adding a couple variations that i could imagine someone hearing word of mouth using just in case.